test_apps/advection/README.md

```diff
@@ -0,0 +1,59 @@
+# VAPOR Particle Advection Benchmarking
+
+This directory has the benchmarking program for VAPOR's particle advection module
+
+## Build project
+
+The project has 1 external dependency apart from VAPOR
+-- The C++ Boost library [Program Options, File System]
+
+In order to find all the required VAPOR components the following variables must
+be provided to CMake
+
+```
+VAPOR_PREFIX : path to the VAPOR build/installation
+VAPOR_INCLUDE_DIR : path to the include directory in the VAPOR repository
+VAPORDEPS_PREFIX : path to where the VAPOR dependencies are located
+```
+A sample CMake command to build this project would look like
+
+```
+cmake -DVAPOR_PREFIX=/Users/abhishek/repositories/VAPOR/build \
```

Comment:
How are the number of threads controlled?

Reply:
I'll add it to the `Running benchmarks` section